mlir_graphblas/mlir_builder.py

```diff
@@ -7,54 +7,125 @@
 ###########
 
 import jinja2
-from collections import OrderedDict
+import regex
+import mlir
+from contextlib import contextmanager
+from collections import OrderedDict, namedtuple
```

Comment:
`namedtuple` is currently not used. I'll remove this.

Reply:
https://github.com/metagraph-dev/mlir-graphblas/pull/47/commits/dfe7d563acf99e1608e2ceb6967e830ce67dc3ce implements these changes.